src/lint/linter/ArcanistESLintLinter.php

```diff
@@ -147,7 +147,15 @@ protected function parseLinterOutput($path, $err, $stdout, $stderr) {
         $messages = array();
         foreach ($results as $result) {
             $ruleId = idx($result, 'ruleId');
-            $description = idx($result, 'message')."\r\nSee documentation at ".$this->getRuleDocumentationURI($ruleId);
+
+            // Only rules built into eslint are guaranteed to have a rule documentation URI.
+            if (strpos($ruleId, '/') !== FALSE) {
```

Comment:
this might be a lot to maintain, but how would you feel about handling some of the plugins we use here? for instance, anything that starts with "react/" could probably be linked to from https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules

So maybe we could have a mapping of plugin to url prefix, and then fall back to not having any message? or is that too much to worry about?